packages/react-core/src/components/Button/Button.tsx

```diff
@@ -94,14 +94,11 @@ export const Button: React.FunctionComponent<ButtonProps & OUIAProps> = ({
       type={isButtonElement ? type : null}
       {...getOUIAProps('Button', ouiaId)}
     >
-      {icon && variant === ButtonVariant.link && iconPosition === 'left' && (
-        <span className={styles.buttonIcon}>{icon}</span>
-      )}
-      {variant === ButtonVariant.link && <span>{children}</span>}
-      {variant !== ButtonVariant.link && children}
-      {icon && variant === ButtonVariant.link && iconPosition === 'right' && (
-        <span className={styles.buttonIcon}>{icon}</span>
+      {icon && iconPosition === 'left' && (
+        <span className={css(styles.buttonIcon, styles.modifiers.start)}>{icon}</span>
       )}
+      {children}
+      {icon && iconPosition === 'right' && <span className={css(styles.buttonIcon, styles.modifiers.end)}>{icon}</span>}
```

Comment:
just curious why this conditional block is formatted differently than the other?

Reply:
Not sure, I think the autoformatter just preferred a new line for the other one. The new update has them formatted the same.